ide/nbproject/project.xml

```diff
@@ -35,6 +35,12 @@
                     <location>${project.dir}/tools/fqltool/src</location>
                     <encoding>UTF-8</encoding>
                 </source-folder>
+                <source-folder>
```

Comment:
TODO: verify this actually works in netbeans.

Reply:
@michaelsembwever todo

Reply:
Ah, forgot to note here, I did verify this works in netbeans (screenshot in pr description)